AK/UFixedBigIntDivision.h

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2023, Dan Klishch <danilklishch@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Diagnostics.h>
+#include <AK/UFixedBigInt.h>
+
+namespace AK {
+
+template<size_t divident_bit_size, size_t divisor_bit_size, bool restore_remainder>
+constexpr void div_mod_internal(
+    StaticStorage<false, divident_bit_size> const& operand1,
+    StaticStorage<false, divisor_bit_size> const& operand2,
+    StaticStorage<false, divident_bit_size>& quotient,
+    StaticStorage<false, divisor_bit_size>& remainder)
+{
+    size_t divident_len = operand1.size(), divisor_len = operand2.size();
+    while (divisor_len > 0 && !operand2[divisor_len - 1])
+        --divisor_len;
+    while (divident_len > 0 && !operand1[divident_len - 1])
+        --divident_len;
+
+    // FIXME: Should raise SIGFPE instead
+    VERIFY(divisor_len); // VERIFY(divisor != 0)
```

Comment:
Why is the second `VERIFY` commented out?
They don't seem to mean the same

Reply:
They indeed mean the same: if `divisor_len == 0`, it means that all of the divisor's words are zeroes, so `divisor == 0`. Not sure if I should update comment with this note.